### PR TITLE
[DOCS-364] Email Alerts for Potential Critical Vulnerabilities

### PR DESCRIPTION
**Pull request by ana-dashuk-cobalt**
### Added

| Page | Deploy Preview | Comment |
| ----- | ----- | ----- |
| Explore Risk Advisories > Subscribe to Email Alerts | https://deploy-preview-276--cobalt-docs.netlify.app/platform-deep-dive/assets/risk-advisories/#subscribe-to-email-alerts |  |

## Preview This Change

To see how this change looks in production, scroll down to **Deploy Preview**. Select the link that looks like `https://deploy-preview-<num>--cobalt-docs.netlify.app/`

## Variables

Help us support a “Write once, publish everywhere” single source of truth. If you see a line that looks like:

`{{% asset-categories %}}`

You’ve found a [shortcode](https://gohugo.io/content-management/shortcodes/) that we include in multiple documents.

You’ll find the content of the shortcode in the following directory:

https://github.com/cobalthq/cobalt-product-public-docs/tree/main/layouts/shortcodes

That shortcode has the same base name as what you see in the PR, such as `asset-categories.html`.

## Checklist for PR Author

[ ] Did you check for broken links and alt text?

Be sure to check for broken links and Alt Text issues. We have a partially automated process,
as described in this section of our repository README:
[Test Links and Alt Attributes](https://github.com/cobalthq/cobalt-product-public-docs/blob/main/README.md#test-links-and-alt-attributes).

### CODE REVIEW

File: content/en/Platform Deep Dive/Assets/risk-advisories.md [hunk 1]
```diff
@@ -83,3 +83,14 @@ Select a vulnerability to view detailed information on the [National Vulnerabili
 {{% alert title="Note" color="primary" %}}
 Potential vulnerabilities on the **Risk Advisory** tab don’t belong to your pentests. These are potential risks based on the <a href="https://www.cve.org/" target="_blank">Common Vulnerabilities and Exposures (CVE)</a> standard that we show for your reference.
 {{% /alert %}}
+
+## Subscribe to Email Alerts
+
+You can subscribe to email alerts for **potential critical vulnerabilities** discovered in your assets. We only send alerts for Critical vulnerabilities, based on the CVSS severity levels.
```
Comment (Ikaneng) [inline content/en/Platform Deep Dive/Assets/risk-advisories.md:89]:
```suggestion
You can subscribe to email alerts for **potential critical vulnerabilities** discovered in your assets. We only send alerts to critical vulnerabilities according to the [CVSS score](https://nvd.nist.gov/vuln-metrics/cvss). That's all vulnerabilities that have a score of 9 or higher.
```

**Review by Ikaneng — APPROVED** [review]

File: content/en/Platform Deep Dive/Assets/risk-advisories.md (hunk 1 above)
Comment (ana-dashuk-cobalt) [reply content/en/Platform Deep Dive/Assets/risk-advisories.md:89]: @Ikaneng Should we also specify the rating version?

My suggestion:

We only send alerts for Critical vulnerabilities with a score of 9 or higher, according to the [NVD CVSS v3.0 ratings](https://nvd.nist.gov/vuln-metrics/cvss).

Comment (Ikaneng) [reply content/en/Platform Deep Dive/Assets/risk-advisories.md]: Yes, your suggestion works for me.

Comment (ana-dashuk-cobalt) [reply content/en/Platform Deep Dive/Assets/risk-advisories.md]: @Ikaneng Thank you, I merged this pull request.

**Pull request merged**